dts/bindings/watchdog/nxp,imx-wdog.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2018, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: imxRT watchdog
```

Comment:
nit: looks like the same wdog on i.mx6/7, so probably good to just say `i.MX watchdog (including i.MX RT1xxx, but not include i.MX- RT6/6xx)`